src/framework/theme/components/autocomplete/autocomplete.directive.ts

```diff
@@ -182,6 +182,9 @@ export class NbAutocompleteDirective<T> implements OnDestroy, AfterViewInit, Con
     return (this.isOpen && this.keyManager.activeItem) ? this.keyManager.activeItem.id : null;
   }
 
+  @HostBinding(`disabled`)
```

Comment:
This binding overrides `disabled` attribute of the input element. Do we need this anyway? `input` element has `disabled` attribute and property why we need to add component binding as well? I think we could stick with `setDisabledState` without additional bindings.